modules/storage-expanding-glusterfs-pvc.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * storage/expanding-persistent-volumes.adoc
+
+[id="expanding-glusterfs-pvc-{context}"]
+= Expanding GlusterFS-based Persistent Volume Claims
```

Comment:
Gluster is not supported in 4.1, perhaps using AWS EBS as example.

Reply:
I've removed this section entirely. I'm not certain having an EBS example would be beneficial, as it would be the exact steps as the section above, just with slightly different names?

If you think it adds value I can include it here.